azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -241,6 +245,12 @@ AlerterHolder getAlerterHolder() {
 
   @Override
   public void run() {
+    // Record the time between submission and when the flow execution starts.
+    // Note that since submit time is recorded on the web server and flow execution happens on
+    // the executor, there could be some inaccuracies due to clock skew.
```

Comment:
Exactly. I am afraid this issue right here renders this metric not very useful. Not sure if there is a better way.

Reply:
I'm not sure how much clock skew we can get if all machines synchronize their time against a server, therefore I'm not sure how much inaccuracy we can expect from this metric, however it can still be useful in my opinion to for example detect anomalies between releases. While single values could be useless due to clock skew it is still valuable to analyze trends over time, right?
Ideally we should use the clock in the DB as the source of truth for this metric but this means modifying current implementations and DB schema to make the DB record operation's timestamps.

Reply:
Talked offline.
Agreed to removing metrics affected by clock skew issues until these are fixed. Deleted metrics:
-`queue-wait-histogram`: Records the time between submission and the start of flow execution preparation.
-`flow-time-to-start-histogram`: Records the time between submission (a new execution request is inserted in the DB) and the start of flow execution.